build/ci/templates/nuget-cache.yml

```diff
@@ -14,8 +14,25 @@ steps:
     displayName: Compress nuget package cache folder
 
   - task: Cache@2
-    condition: eq(variables['enable_package_cache'], 'true')
+    condition: eq(variables['enable_azdo_package_cache'], 'true')
     inputs:
       key: nuget | $(Agent.OS) | $(Agent.JobName) | $(build.sourcesdirectory)/**/*.csproj | $(build.sourcesdirectory)/**/Directory.Build.targets | $(build.sourcesdirectory)/**/Directory.Build.props
       path: ${{ parameters.nugetPackages }}
     displayName: Restore NuGet packages cache
+
+  - task: Cache@2
+    condition: eq(variables['enable_azdo_package_cache'], 'true')
+    inputs:
+      key: nuget | $(Agent.OS) | $(Agent.JobName) | $(build.sourcesdirectory)/**/*.csproj | $(build.sourcesdirectory)/**/Directory.Build.targets | $(build.sourcesdirectory)/**/Directory.Build.props
```

Comment:
I guess nuget.config should be included as cache key.

Reply:
This caching will be removed in the future as it does not work. Even this overall PR shows that caching nuget content is not helping much (if at all) the build time.